acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -6,11 +6,12 @@ module Puppet
   module Acceptance
     module InstallUtils
       PLATFORM_PATTERNS = {
-        :redhat        => /fedora|el|centos/,
         :debian        => /debian|ubuntu/,
         :debian_ruby18 => /debian|ubuntu-lucid|ubuntu-precise/,
+        :redhat        => /fedora|el|centos/,
+        :sles          => /sles/,
         :solaris       => /solaris/,
-        :windows       => /windows/,
+        :windows       => /windows/
```

Comment:
rubocop complains, so i guess at some point i cleaned it out of there.  i
also like your reasoning.  i should probably turn that rule off

On Wed, Feb 25, 2015 at 2:15 PM, Kylo Ginsberg <notifications@github.com>
wrote:

> In acceptance/lib/puppet/acceptance/install_utils.rb
> <https://github.com/puppetlabs/facter/pull/856#discussion_r25385005>:
>
> >          :solaris       => /solaris/,
> > -        :windows       => /windows/,
> > +        :windows       => /windows/
>
> Was this for style reasons or was there breakage?
>
> FWIW, I always like the last element to have a trailing , (when the
> language permits it) so that future additions have a cleaner diff. But
> that's strictly preference.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/facter/pull/856/files#r25385005>.
>



-- 
Eric Thompson
eric.thompson@puppetlabs.com <erict@puppetlabs.com>
Senior Software Quality Assurance Engineer